go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/submariner-io/submariner
 
 go 1.21
+toolchain go1.22.2
```

Comment:
It's interesting this toolchain directive gets introduced by this PR. I'm a bit surprised, as we already had golang 1.21 here which introduced the feature.

Reply:
K8s 0.30 requires Go 1.22. We should just update to `go 1.22` and remove the toolchain directive.

Reply:
See https://github.com/submariner-io/shipyard/pull/1620